src/Plugin.Maui.Audio/AudioPlayer.net.cs

```diff
@@ -29,4 +29,12 @@ partial class AudioPlayer : IAudioPlayer
 	public void Stop() { }
 
 	public void Seek(double position) { }
+
+	public double Speed { get; set; }
+
+	public double MinimumSpeed { get; }
+
+	public double MaximumSpeed { get; set; }
```

Comment:
Do we need the setter?

Reply:
If not then here's a 'quick fix'

```suggestion
	public double MaximumSpeed { get; }
```

Reply:
Oh no definitely not, probably some muscle memory or something going on here, thanks :D